From 59cefe2ee9b4785bd52960a492dc579d8f38eb86 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Miro=20Hron=C4=8Dok?= <miro@hroncok.cz>
Date: Tue, 29 Jul 2025 23:23:15 +0200
Subject: [PATCH] Do not require wheel for building

 - current version of setuptools (70.1+) does not need wheel at all
 - older versions of setuptools would fetch wheel when building wheels (but not sdists)
---
 pyproject.toml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 551da23..2e3377c 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,7 +1,6 @@
 [build-system]
 requires = [
     'setuptools>=44',
-    'wheel>=0.30.0',
 ]
 build-backend = 'setuptools.build_meta'
 
